src/apix-components/apix.model.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export class Api {
```

Comment:
there should be comments in this file about the fact that these are model classes that reflect the values returned by the code.vmware.com APIX services.

Reply:
done